libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts

```diff
@@ -0,0 +1,16 @@
+import { Route } from '@angular/router';
+
+export const adminRoutes: Route[] = [
+  {
+    path: '',
+    loadChildren: () =>
+      import('./dashboard/dashboard.routes').then((m) => m.dashboardRoutes),
```

Comment:
you can use the default keyword to avoid `then`inside your promise

Reply:
ya that will be much cleaner as well, thanks i will implement it from now